includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -6,6 +6,7 @@ class DesignSystemSharedLinks {
 	private static $instance;
 
 	private function __construct() {
+		$this->mapHrefsToLocalUrls();
```

Comment:
What do you think about modifying urls in `getHref` method instead of replacing all url on object initialization?